### PR TITLE
Fix convert2rhel test

### PR DESCRIPTION
**Pull request by shweta83**
### Problem Statement
Convert2Rhel tests failing because of outdated packages.

### Solution
Updated packages before running convert2rhel job on host

### CODE REVIEW

**Comment by shweta83** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_convert2rhel.py -k test_convert2rhel_centos

**Comment by synkd** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_convert2rhel.py -k test_convert2rhel_oracle

**Comment by shweta83** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_convert2rhel.py

**Comment by shweta83** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_convert2rhel.py -k test_convert2rhel_oracle[oracle8]

**Comment by shweta83** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_convert2rhel.py

**Comment by shweta83** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_convert2rhel.py

**Comment by shweta83** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_convert2rhel.py

File: tests/foreman/api/test_convert2rhel.py [hunk 1]
```diff
@@ -199,12 +200,17 @@ def oracle(
...
     result = oracle_host.execute(
         'yum install -y kernel && '
         'grubby --set-default /boot/vmlinuz-'
         '`rpm -q --qf "%{BUILDTIME}\t%{EVR}.%{ARCH}\n" kernel | sort -nr | head -1 | cut -f2`'
     )
     assert result.status == 0
+    oracle_host.execute(
+        'sed -i -- \'s/CleanupModulesOnExit=yes/CleanupModulesOnExit=no/g\'",'
+        ' "/etc/firewalld/firewalld.conf"'
+    )
```
[the remark this answers is not in the document]
Comment (shweta83) [reply tests/foreman/api/test_convert2rhel.py]: @Gauravtalreja1 These fixes were not working in the  fixture. When fix these after host reboot, then it's working. I don't know the reason behind it but you can update the other PR and we will merge it. Then I can rebase my PR. Whatever works for you. Let me know and let's get this merged.

[the remark this answers is not in the document]
Comment (Gauravtalreja1) [reply tests/foreman/api/test_convert2rhel.py]: Okay, I'll update 13236 first, where only oracle8 test fails for conversion due to known inhibitor, which we can debug later, but I'd suggest to merge 13236 first, then we can revisit this PR until then we can move this to draft

**Comment by shweta83** [comment]
Closing this PR as the changes are addressed in another PR https://github.com/SatelliteQE/robottelo/pull/14194

**Pull request closed without merging**